src/treemodel/selection.js

```diff
@@ -99,17 +106,20 @@ export default class Selection {
 
 	/**
 	 * Adds a range to the selection. Added range is copied and converted to {@link core.treeModel.LiveRange}. This means
-	 * that passed range is not saved in the Selection instance and you can safely operate on it. Accepts a flag
-	 * describing in which way the selection is made - passed range might be selected from {@link core.treeModel.Range#start}
-	 * to {@link core.treeModel.Range#end} or from {@link core.treeModel.Range#start} to {@link core.treeModel.Range#end}. The flag
-	 * is used to set {@link #anchor} and {@link #focus} properties.
+	 * that passed range is not saved in the Selection instance and you can safely operate on it.
 	 *
+	 * Accepts a flag describing in which way the selection is made - passed range might be selected from
+	 * {@link core.treeModel.Range#start} to {@link core.treeModel.Range#end} or from {@link core.treeModel.Range#start}
```

Comment:
This sentence makes no sense. I guess the latter selection should be from `#end` to `#start`.